chunk_test.go

```diff
@@ -611,3 +611,27 @@ func TestQueryArrowStream(t *testing.T) {
 		}
 	})
 }
+
+func TestQueryArrowStreamDescribeOnly(t *testing.T) {
+	runSnowflakeConnTest(t, func(sct *SCTest) {
+		numrows := 50000 // approximately 10 ArrowBatch objects
+
+		query := fmt.Sprintf(selectRandomGenerator, numrows)
+		loader, err := sct.sc.QueryArrowStream(WithDescribeOnly(sct.sc.ctx), query)
+		if err != nil {
+			t.Error(err)
```

Comment:
IMO `t.Fatal` is better as we shouldn't proceed with the test when error occurs?